zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/ProviderRegistry.java

```diff
@@ -29,8 +29,10 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ProviderRegistry.class);
 
+    public static final String AUTHPROVIDER_PROPERTY_PREFIX = "zookeeper.authProvider.";
```

Comment:
nit:

I like that you created this public constant. I searched for this, and the same string is also used in `QuorumPeerConfig`:

https://github.com/apache/zookeeper/blob/c42c8c94085ed1d94a22158fbdfe2945118a82bc/zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java#L508

Can you use `AUTHPROVIDER_PROPERTY_PREFIX` there too?

Reply:
Sure, will do. Looks like there are a lot of unit tests where we use the same string, but I won't replace all of them in this PR.